airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -652,13 +652,14 @@ class TicketForms(SourceZendeskSupportCursorPaginationStream):
 
 class TicketMetrics(SourceZendeskSupportCursorPaginationStream):
     """TicketMetric stream: https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_metrics/"""
-
+    page_size = 100
+    
     def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
         """
         https://developer.zendesk.com/documentation/api-basics/pagination/paginating-through-lists-using-cursor-pagination/#when-to-stop-paginating
         """
         meta = response.json().get("meta", {})
-        return meta.get("after_cursor") if meta.get("has_more", False) else None
+        return meta.get("after_cursor") if meta.get("has_more", True) else None
```

Comment:
Can you explain how the change you're doing fix the issue?

Reply:
I believe `meta.get("has_more", False)` will return `False` if the `meta` dict does not have a `has_more` key in it. This will also yield the same result if `meta` has a key `has_more` with the value `False`, which is the expected value when pagination limit is reached.

What that means is that the return value of `next_page_token` will be `meta.get("after_cursor")` only when `has_more` is `True`

Reply:
This logic makes sense to me @btkcodedev, we only need `after_cursor` to be returned when `has_more` is `True` (and there are more records to fetch)

Zendesk has a [pagination example](https://developer.zendesk.com/documentation/api-basics/pagination/paginating-through-lists-using-cursor-pagination/#python-example-with-the-after-cursor) that affirms us:

```
    if data['meta']['has_more']:
        params['page[after]'] = data['meta']['after_cursor']
    else:
        url = None
```

Reply:
I checked the [python docs](https://docs.python.org/3/library/stdtypes.html#dict.get) and it turns out `if meta.get("has_more", False)` actually just sets False as default, if `has_more` has a value then it will be fetched as is. 👍🏽